doc/source/user_guide/fields/field_data.rst

```diff
@@ -89,7 +89,7 @@ To obtain face connectivity data, specify ``FacesConnectivity`` as the ``data_ty
 
   # FacesConnectivity provides indices of vertices for each face. For example:
   # Face 6 is connected to vertices 4, 5, 12, and 11.
-  >>> faces_connectivity_data["inlet"][SurfaceDataType.FacesConnectivity][5]
+  >>> faces_connectivity_data["inlet"].connectivity[5]
```

Comment:
@prmukherj As per this example, connectivity is an array of integer arrays, so the typehint is not correct. Same issue with lines.

Reply:
Thank you, Done.